### PR TITLE
[Test][Doc] Messages for Health API

### PR DESCRIPTION
**Pull request by karenzone**
### DO NOT MERGE

Testing ideas for delivery, presentation, and formatting for Health API messages

**PREVIEW:**  https://logstash_bk_16508.docs-preview.app.elstc.co/guide/en/logstash/master/ts-health-api-msgs.html

### CODE REVIEW

**Comment by karenzone** [comment]
@yaauie, thanks for the conversation today.  You know that I can't resist an interesting docs challenge.  
We kicked around some interesting workarounds today that involved pushing the capabilities of our docs system and might create some docs system migration problems. 

Now I'm noodling on a simpler approach that could possibly bring additional value and context to our users. I was originally picturing having one error message per page.  Now I'm leaning toward grouping messages so that the information experience doesn't seem so disjointed, and also provides additional context relative to other error conditions. 

Each topic (in this case, error message) will have its own ID, so we can still deep link to highlight the specific message. 

I have this sample building locally, and will post a link after the preview is generated.

**Comment by karenzone** [comment]
I was playing around locally, and now I'm wishing I'd picked more compelling titles and wording.  But this demo should at least convey the mechanics.

**Comment by yaauie** [comment]
Fantastic. I'll plug my text in and continue to iterate.

File: docs/static/troubleshoot/troubleshooting.asciidoc [hunk 1]
```diff
@@ -28,3 +28,4 @@ include::ts-logstash.asciidoc[]
 include::ts-plugins-general.asciidoc[]
 include::ts-plugins.asciidoc[]
 include::ts-other-issues.asciidoc[]
+include::ts-health-api-msgs.asciidoc[]
```
Comment (karenzone) [inline docs/static/troubleshoot/troubleshooting.asciidoc:31]: Structure pulled into another body of work

**Pull request closed without merging**